### DISCUSSION TITLE
FR: Dashboard Widget - Upcoming Plugin Subscription Renewals

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8678] [Ideas]
It would be great if Craft came with a default dashboard widget to show the next “x” upcoming plugin subscription renewals - I.e. the plugin name, date of renewal and colour/icon to indicate if “auto renewal” was enabled. It would be awesome if the renewal $ amount was also listed. 

Currently I can’t see anywhere in the control panel to see any of this info. I have to login to id.craftcms to see the expiration dates.

**Comment by angrybrad** [comment]
Cool idea.

**Comment by brandonkelly** [comment]
Related to #4247
